bitnami/discourse/3/debian-12/rootfs/opt/bitnami/scripts/discourse-env.sh

```diff
@@ -172,6 +174,8 @@ DISCOURSE_REDIS_PORT_NUMBER="${DISCOURSE_REDIS_PORT_NUMBER:-"${REDIS_PORT_NUMBER
 export DISCOURSE_REDIS_PORT_NUMBER="${DISCOURSE_REDIS_PORT_NUMBER:-6379}"
 DISCOURSE_REDIS_PASSWORD="${DISCOURSE_REDIS_PASSWORD:-"${REDIS_PASSWORD:-}"}"
 export DISCOURSE_REDIS_PASSWORD="${DISCOURSE_REDIS_PASSWORD:-}"
+DISCOURSE_REDIS_DB="${DISCOURSE_REDIS_DB:-"${REDIS_DB:-}"}"
+export DISCOURSE_REDIS_DB="${DISCOURSE_REDIS_DB:-}"
```

Comment:
Can you force the default value here to be `0` and also add a new validation into `libdiscourse.sh`? There is a common `is_positive_int()` method that can be of help.

Reply:
Sure, will do it.

Reply:
Done